class/Validation.php

```diff
@@ -0,0 +1,544 @@
+<?php
+
+require_once('config/error_messages.php');
+require_once('class/DatabaseConnection.php');
+require_once('config/constants.php');
```

Comment:
Constant file is already included.
Also you are not using any constants in this file.